config/tizen/gbsbuild.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "******************************************************************"
+echo "*                       Tizen GBS build                          *"
+echo "*                                                                *"
+echo "* Please input user, passwd of http://Tizen.org on '~/.gbs.conf' *"
+echo "* For more information, please read Guide Docs. folder           *"
+echo "* ~/.gbs.conf sample is at 'config/tizen/sample.gbs.conf'.       *"
+echo "*                                                                *"
+echo "******************************************************************"
+read -p "[Warning] This working folder will be copied to ../iotjs_tizen_gbs \
+Are you sure to continue? (y/n) " -n 1 -r
+cd ..
+cp -ra iotjs iotjs_tizen_gbs
```

Comment:
We can't assume that the previous working directory is always iotjs. So I suggest using `cp -ra $OLDPWD iotjs_tizen_gbs`